### ISSUE TITLE
Write a vale rule to check that all images have alt text

### ISSUE THREAD

**Issue by TibsAtWork** [aiven/devportal#1130]
We always want our images to have alt text, but it's very easy to forget.

Until recently, leaving out an `:alt:` value on a `.. image::` directive would cause a weird spelling error later in the document, which meant we tended to find them. But that problem appears to have been fixed.

It's not clear whether this check will need to look at the "raw" markup.

**Comment by TibsAtWork** [comment]
Warning: today I've seen a build in CI that still reported a spelling error when the alt text is missing, albeit actually localised to the image directive

<img width="670" alt="Screenshot_2022-07-13_at_10 52 29" src="https://user-images.githubusercontent.com/42539688/178706966-3c9843f5-7a62-4d24-8fa4-8c38cbc990ce.png">

Interestingly it *doesn't* show up if I do `make spell` locally, where I definitely have vale 2.20.0, so perhaps the CI has not updated itself yet.

**Comment by TibsAtWork** [comment]
We're using `master` of vale-action, but unfortunately that's still using vale 2.15.5, and the bug doesn't go away until vale 2.18.0

I was being confused by the "action at a distance" where vale-action says what version of vale to use 😢 

Since I'm not going to be working on this very soon, it's probably enough for now to wait for vale-action to be updated - I can always raise an issue on it if we're in more of a hurry.

**Comment by TibsAtWork** [comment]
https://github.com/aiven/devportal/issues/1235 and https://github.com/aiven/devportal/pull/1241 should move us to using the latest version of Vale.

**Comment by TibsAtWork** [comment]
A test to make sure all `.. image::` directives have `:alt:` text was added in https://github.com/aiven/devportal/pull/1370, so this issue can be closed.
